assets/app/view/game/tile.rb

```diff
@@ -48,7 +48,7 @@ def render
         children = []
 
         render_revenue = should_render_revenue?
-        children << render_tile_part(Part::Track, routes: @routes) if @tile.paths.any?
+        children << render_tile_part(Part::Track, routes: @routes) if (@tile.paths + @tile.path_stubs).any?
```

Comment:
since this is a bit expensive i wonder if it's worth to do 

@tile.paths.any || tile.path_stubs.any?